.gitignore

```diff
@@ -12,3 +12,5 @@ profiles.clj
 .idea
 club_of_naaarfs.iml
 club_of_naaarfs_dev.db.mv.db
+club_of_naaarfs_dev.db.trace.db
```

Comment:
we should exclude 
```
*.mv.db
*.trace.db
```

as everybody uses its own db name in profiles.clj or ``DATABASE_URL``  env var